bin/spark-submit

```diff
@@ -19,6 +19,9 @@
 
 SPARK_HOME="$(cd "`dirname "$0"`"/..; pwd)"
 
+# disable randomized hash for string in Python 3.3+
```

Comment:
I guess we need this so that all Python workers produce the same hash for the same string (e.g. for hash portability).  Nice catch.